packages/atlas/src/components/Tooltip/Tooltip.tsx

```diff
@@ -43,13 +43,35 @@ export const Tooltip: React.FC<TooltipProps> = ({
   delay,
   customContent,
   showOnCreate,
-  oneLine,
+  multiline,
   className,
 }) => {
   const [isVisible, setIsVisible] = useState(false)
+
+  const content = customContent ? (
```

Comment:
Wouldn't this to the job?
```suggestion
const content = customContent || (
    <TooltipContent headerText={!!headerText}>
      <TooltipHeader headerText={!!headerText}>
        {icon && (
          <IconWrapper>
            <StyledSvgAlertsInformative24 />
          </IconWrapper>
        )}
        {headerText && <TooltipText variant="h100">{headerText}</TooltipText>}
      </TooltipHeader>
      {text && text.length && (
        <TooltipText withIcon={!!icon} headerText={!!headerText} variant="t100">
          {text}
        </TooltipText>
      )}
    </TooltipContent>
  )
```

Reply:
It's the same thing really